### DISCUSSION TITLE
Set the default level for different paths with `tracing-appender`

### DISCUSSION THREAD

**Discussion by yatesco** [tokio-rs/tracing#1775] [Q&A] [answered]
Hi there, is there a way, using the `tracing-appender` of saying "I want `file/a/b` to be `INFO` but all else to be `ERROR`, similar to `env-logger` or `log4rs`?

I guess this is a question for the `tracing_subscriber` but I can quite see how it fits together.

Thanks!

**Comment by rminderhoud** [comment]
Old question but this is possible by using `with_max_level` on the writer

**Reply by davidbarsky** [reply, marked as answer]
I somehow missed this this question initially. Sorry about that, @yatesco. While @rminderhoud is correct, we generally recommend using multiple layers (see [this example](https://github.com/tokio-rs/tracing/blob/v0.1.x/examples/examples/fmt-multiple-writers.rs)) alongside [per-Layer filters](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/index.html#filtering-with-layers).
